src/config/app_production_dependency.go

```diff
@@ -1,29 +0,0 @@
-package config
-
-import (
-	"github.com/Originate/exosphere/src/types"
-)
-
-// AppProductionDependency contains methods that return config information about a dependency
-type AppProductionDependency interface {
-	HasDockerConfig() bool
-	GetDockerConfig() (types.DockerConfig, error)
-	GetServiceName() string
-	GetDeploymentConfig() (map[string]string, error)
-	GetDeploymentServiceEnvVariables(secrets types.Secrets) map[string]string
-	GetDeploymentVariables() (map[string]string, error)
-}
-
-// NewAppProductionDependency returns an AppProductionDependency
-func NewAppProductionDependency(dependency types.ProductionDependencyConfig, appContext *types.AppContext) AppProductionDependency {
-	switch dependency.Name {
-	case "exocom":
-		return &exocomProductionDependency{dependency, appContext}
-	case "postgres":
-		fallthrough
-	case "mysql":
-		return &rdsProductionDependency{dependency, appContext}
-	default:
-		return &genericProductionDependency{dependency}
-	}
-}
```

Comment:
Renamed a lot of these from `app_production_dependency.go` -> `remote_app_dependency.go`, or `exocom_development_dependency.go` -> `local_exocom_dependency.go`